src/Android/Xamarin.Android/Samples/Security/OAuth/readme.md

```diff
@@ -20,24 +14,25 @@ When you run the sample, the app will load a web map which contains premium cont
 
 ## How it works
 
-1. Set the `AuthenticationManager`'s `AuthenticationChallengeHandler` to a `DefaultAuthenticationChallengeHandler`.
-2. Create an `OAuthConfiguration` specifying the portal URL, client ID, and redirect URL.
-3. Add the OAuth configuration to the authentication manager.
-4. Load a map with premium content requiring authentication to automatically invoke the default authentication handler.
+1. Set the `AuthenticationManager`'s `ChallengeHandler`.
```

Comment:
I'm guessing this is actually necessary (i.e. .NET doesn't wire up a default challenge handler automatically)?

Reply:
Just checked, it is necessary for a proper workflow. You have to login twice without it.

Reply:
Ok. Thanks for checking. Looks good then. Thanks for bearing with me!